tracex_project/extraction/views.py

```diff
@@ -130,7 +131,7 @@ def form_valid(self, form):
             return render(
                 self.request,
                 "error_page.html",
-                {"type": type(e).__name__, "error_traceback": traceback.format_exc()}
+                {"type": type(e).__name__, "error_traceback": traceback.format_exc()},
```

Comment:
why is that , needed?

Reply:
i wouldnt say its necessary either and i dont really remember adding it. Maybe it was auto-added via the precommit hook or formatting the file 🤷‍♂️ i can remove it though

Reply:
As commas are so common in Python, the comma is IMO a bestpractice to enable a new addition without having to set a comma in the last line